stacks/container-apps/cms.prx.org.yml

```diff
@@ -379,6 +384,58 @@ Resources:
         - Key: "prx:cloudformation:stack-id"
           Value: !Ref AWS::StackId
       TaskDefinition: !Ref WorkerTaskDefinition
+
+  # memcached
+  CmsMemcacheSubnetGroup:
+    Type: AWS::ElastiCache::SubnetGroup
+    Properties:
+      Description: !Sub CMS ${EnvironmentType} memcache subnet group
+      SubnetIds:
+        - !Ref VPCSubnet1
+        - !Ref VPCSubnet2
+  CmsMemcacheSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      VpcId: !Ref VPC
+      GroupDescription: !Sub CMS ${EnvironmentType} memcached security group
+      SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
```

Comment:
Open to the world?  Or I guess just the VPC.

Reply:
Copied what was in the memcache template that the existing cluster was launched from. If there's a more locked-down way to do this, especially now that it's in the same VPC, I'd be up for that, I'm just bad at VPCs.